01/app.js

```diff
@@ -1,2 +1,10 @@
 const randomNumber = Math.round(Math.random() * 20);
-console.log(randomNumber);
+console.log('Wylosowana liczba: ' + randomNumber);
+let j=randomNumber;
```

Comment:
nie rozumiem koncepcji tego przypisania.
Jak już to możesz to zrobić w pętli tj. `for(let j=randomNumber`